OpenSim/Common/LogManager.cpp

```diff
@@ -102,15 +102,17 @@ sync()
 LogManager::LogManager()
 {
     // Seems to be causing crashes in the GUI... maybe a multithreading issue.
-#if 1
+
     // Change the underlying streambuf for the standard cout/cerr to our custom buffers
     std::cout.rdbuf(&out);
     std::cerr.rdbuf(&err);
 
-    // Example setup: redirect output to both the terminal and an output file
+    // Redirect output to both the terminal and an output file
     out.addLogCallback(new StreamLogCallback(&cout,false));
```

Comment:
The wording in the comment 'both' is a bit misleading since it's conditional now.